style.css

```diff
@@ -0,0 +1,104 @@
+html {
+    height: 100%;
+    --cta-light: #fbcfe8;
+    --border: #e5f0f0;
+}
+
+.hero-sec {
+    width: 100%;
+    height: 100%;
+    padding: 0;
+}
+
+.hero-sec>img {
+    width: 90%;
+    margin: 2rem 4rem;
+    object-fit: fill;
+    object-position: top right;
+}
+
+.categories {
+    width: 70%;
+    margin: 1rem auto;
+    display: flex;
+    justify-content: space-around;
+}
+
+.categories>img {
+    border: 2px solid black;
+    height: 150px;
+    object-fit: contain;
+}
+
+.text-over {
+    background-color: var(--cta-light);
+    color: var(--cta);
+    width: 100%;
+    position: relative;
+    top: -5rem;
+    text-align: center;
+}
+
+.categories-card {
+    border: 2px solid var(--border);
+    /* box-shadow:
```

Comment:
we can remove unused CSS styling.

Reply:
Thank you for pointing it out abhishek , i have made the changes will merge it